tools/binlog/deploy.md

```diff
@@ -0,0 +1,488 @@
+---
+title: TiDB-Binlog Cluster 部署
+category: tools
+---
+
+
+# TiDB-Binlog 部署
+
```

Comment:
@GregoryIan @WangXiangUSTC  I think we might need to add some description about the Kafka version and add the link here.

Reply:
We are deliberately hiding the kafka version